app.py

```diff
@@ -55,6 +65,20 @@ def post_something():
             "ERROR": "no name found, please send a name."
         })
 
+@app.route('/get/', methods=['GET'])
+def generate_session_id():
+    session_id = str(uuid.uuid4())
+    param = request.form.get('username')
+    return session_id
+
+@app.route('/post/', methods=['POST'])
```

Comment:
Likewise.

Reply:
Move to googleapi.py(app.py only concerns with interacting with FE, but this method receives restaurant info from the Google Api, not the user).